frameworks/digital-outcomes-and-specialists/questions/services/outcomesApproachQuestion.yml

```diff
@@ -0,0 +1,5 @@
+type: text
+question: A question for outcomes
+depends:
+  - "on": lot
+    being: ["digital-outcomes"]
```

Comment:
We generally use YAML-onic syntax for arrays, even if they only have one item, eg

``` YAML
depends:
  on: lot
  being:
    - digital-outcomes
```